backend/data/src/main/kotlin/io/tolgee/service/export/ExportFilePathProvider.kt

```diff
@@ -18,6 +18,14 @@ class ExportFilePathProvider(
     languageTag: String? = null,
     replaceExtension: Boolean = true,
   ): String {
+    // If no template is provided, use a default template for xcstrings
+    if (params.fileStructureTemplate == null) {
```

Comment:
This should not happen on this level. This is generic file used for all the formats. Default templates are defined here:

io/tolgee/formats/ExportFormat.kt:8